### ISSUE TITLE
维度错误

### ISSUE THREAD

**Issue by YJYTJ** [SJTU-Quant/SNEX-GNN#2]
您好！我在运行代码时在models/sunnygnn.py的第82行sim = torch.exp(torch.cosine_similarity(anchor[i], aug, dim=2) / self.tau)遇到报错Dimension out of range (expected to be in range of [-2, 1], but got 2)，使得代码无法运行成功。请问如何改正？因为想使用您的代码作为对比实验之一，所以希望尽快得到您的回复！非常感谢！
